sys/linux/dev_binder_386.const

```diff
@@ -32,8 +32,10 @@ BINDER_TYPE_WEAK_HANDLE = 2003315333
 BINDER_WRITE_READ = 3224396289
 FLAT_BINDER_FLAG_ACCEPTS_FDS = 256
 FLAT_BINDER_FLAG_TXN_SECURITY_CTX = 4096
+# MAP_SHARED is not set
 O_NONBLOCK = 2048
 O_RDWR = 2
+# PROT_READ is not set
 TF_ACCEPT_FDS = 16
```

Comment:
@dvyukov It seems that macro MAP_SHARED and PROT_READ is used without including their header in dev_binder.txt

Reply:
Yes. You may either add it, or we can merge it as is. Values for MAP_SHARED/PROT_READ will be taken from other .const files, so it's totally broken.

Reply:
It is the reason of "non sendmsg related changes" in this PR. So, your meaning is keeping the dev_binder.*.const as it was, right?

Reply:
Either way is fine in this case: revert or keep as it is now, does not matter much.

Reply:
Is there anythings should i modify?

Reply:
This place is fine as is.